assets/css/styles.css

```
@@ -50,6 +50,15 @@ h5 a:hover {
   color: #045c64;
 }
 
+.open-science-content h1 {
+    margin-bottom: 2rem;
+}
+
+.open-science-content h2,
+.open-science-content h3 {
```

As @senthil10 said, h4-6 were not included. I think it looks better if they are
```suggestion
.open-science-content h3,
.open-science-content h4,
.open-science-content h5,
.open-science-content h6 {
```